docs/installation.rst

```diff
@@ -1,13 +1,101 @@
 Installing Metagov
 ==================
 
-This documentation will walk through how to set up Metagov on an Ubuntu server.
-If you want to set up Metagov for local development, follow the instructions at :doc:`Local Development <../development>` instead.
+This documentation will walk through how to set up Metagov to work with a Driver. If you want to set up Metagov for local development, follow the instructions at :doc:`Local Development <../development>` instead.
 
-Metagov needs to be installed on the same server as your governance "Driver":
+There are three ways to work with Metagov:
 
-* If you're using PolicyKit as your Driver, head over to the `PolicyKit Documentation <https://policykit.readthedocs.io/>`_ for instructions on how to install PolicyKit on your server.
-* If you're using Metagov alongside your existing system, just make sure that you're installing Metagov on the same server. This is necessary because Metagov and your system will communicate over the local network.
+1. Import the Django app directly into a Django-based Driver.
+2. Communicate with a hosted version of Metagov via public (authenticated) HTTP endpoints.
+3. Install your own version of Metagov and communicate locally via HTTP.
+
+Instructions for #1 and #3 are written below. We are in the process of launching a hosted version of Metagov (#2). Once that's set up, documentation on how to work with it will be linked here.
+
+Using Metagov with a Django-based Driver
```

Comment:
I think the missing section is how to use the Metagov App?
We could include a snippet like:
```
from metagov.core.app import MetagovApp
from metagov.core.handlers import MetagovRequestHandler

metagov = MetagovApp()
metagov_handler = MetagovRequestHandler(metagov)
```

And we should add autodocs for the `MetagovApp` class at some point